modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/SuggestedFolder.kt

```diff
@@ -0,0 +1,16 @@
+package au.com.shiftyjelly.pocketcasts.models.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+
+@Entity(
+    tableName = "suggested_folders",
+    indices = [Index(value = ["folder_name", "podcastUuid"], unique = true)],
```

Comment:
What's the purpose of this index?

Reply:
I wanted to ensure that I have only one Podcast A in an folder, but since I am cleaning the folders before adding new ones I don't need this anymore. I will remove it

Reply:
Ah, makes sense. Didn't thought of it. But yeah, If you clear the DB and if you change the primary key to be defined like this index there shouldn't be a need for it.